scripts/build.sh

```diff
@@ -86,17 +32,9 @@ for TARGET_NAME in "$SERVER_TARGET_NAME" "$CLIENT_TARGET_NAME"; do
         CMD_PATH="${GO_PROJECT}/cmd/client"
     fi
 
-    if [[ "$BUILDER_IMAGE" == "none" ]]; then
-        GOOS="$TARGET_OS" GOARCH="$TARGET_ARCH" CGO_ENABLED=0 go build -v -ldflags \
-            "-X $GO_PROJECT/version.Version=$VERSION" \
-            -o ${TARGET_NAME} ${CMD_PATH}
-    else
-        docker run --rm --privileged -it -v $(pwd):/${TARGET_NAME} -w /${TARGET_NAME} \
-            -e GOOS="$TARGET_OS" -e GOARCH="$TARGET_ARCH" -e CGO_ENABLED=0 \
-            $BUILDER_IMAGE go build -v -ldflags \
-            "-X $GO_PROJECT/version.Version=$VERSION" \
-            -o /${TARGET_NAME}/${TARGET_NAME} ${CMD_PATH}
-    fi
+    GOOS="$TARGET_OS" GOARCH="$TARGET_ARCH" CGO_ENABLED=0 go build -v -ldflags \
```

Comment:
seems TARGET_OS and TARGET_ARCH are undefined now

Reply:
Yes, I forgot to remove then from the build command.